Schema/Player.txt

```diff
@@ -0,0 +1,30 @@
+PlayerId
+Appearance
+FirstName
+LastName
+
+Assists
+AssistDefense (defensive awareness)
+
+Blocks
+BlockDefense (creativity, elusiveness)
+
+Fouls
+FoulDefense (ability to draw fouls)
+
+Rebounds (offensive rebounding)
+ReboundDefense (defensive rebounding)
+
+Shots
+ShotDefense (on ball defense)
+
+Steals
+StealDefense (ballhandling)
+
+ThreePointShots
+
+Turnovers
+TurnoverDefense (ballhandling 2)
+
+ThreePointLikelihood (2 pt vs 3 pt shooting - range)
+
```

Comment:
this is fine, but it should probably move to the BBCentral wiki I think since this is likely to be of more use as human readable information than any kind of code documentation. If this is intended to be true code/schema documentation, then we should delete it. We need the code, schema, &/or APIs to be self documenting. Otherwise these docs will fall out of repair and will start to tell falsehoods. Anyway, we can leave this, but we should think about moving it.